File: app/src/main/java/com/horizon/lightkvdemo/config/GlobalConfig.java
package com.horizon.lightkvdemo.config;


import android.content.Context;

import com.horizon.lightkvdemo.BuildConfig;

public class GlobalConfig {
    public static final boolean DEBUG = BuildConfig.DEBUG;

    private static Context sAppContext;

    public static void setAppContext(Context context) {
        sAppContext = context;
    }

    public static Context getAppContext() {
        return sAppContext;
    }
}

File: app/src/main/java/com/horizon/lightkvdemo/config/GlobalConfig.kt
package com.horizon.lightkvdemo.config


import android.annotation.SuppressLint
import android.content.Context

import com.horizon.lightkvdemo.BuildConfig

@SuppressLint("StaticFieldLeak")
object GlobalConfig {
    val DEBUG = BuildConfig.DEBUG

    var appContext: Context? = null
}
